onnx/cpp2py_export.cc

```diff
@@ -469,7 +469,9 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
       .def(
           "get_all_schemas_with_history",
           []() -> const std::vector<OpSchema> { return OpSchemaRegistry::get_all_schemas_with_history(); },
-          "Return the schema of all existing operators and all versions.");
+          "Return the schema of all existing operators and all versions.")
+      .def(
+          "register_schema", [](OpSchema* op) { RegisterSchema(*op); }, "Register the custom OpSchema.");
```

Comment:
```suggestion
          "register_schema", [](OpSchema* schema) { RegisterSchema(*schema); }, "Register a user provided OpSchema.");
```

nit: wording and naming